scripts/build.sh

```diff
@@ -21,9 +21,6 @@ echo "Executing in $(pwd)"
 echo "Description: Build the package in build/"
 echo "====================================================="
 
-echo "Installing Dependencies"
-npm install > /dev/null
```

Comment:
Should we let `npm install` be a part of build.sh only, as we need to do `npm install` separately everywhere?

Reply:
The problem is that running lint/prettier also requires `npm install`. So if we added this step inside `build.sh`, you will only be able to lint AFTER the build, but we want to fail fast. In package.json, we are doing this now

````
"build": "npm install && npm run lint && npm run pretty && npm run build-ci",
```